src/figuresClasses.ts

```diff
@@ -1,11 +1,74 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = 'triangle',
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
```

Comment:
The variables `a`, `b`, and `c` should be accessed using `this.a`, `this.b`, and `this.c` inside the constructor to refer to the instance properties.